uui/components/tables/DataTableCell.tsx

```diff
@@ -1,16 +1,21 @@
 import * as React from 'react';
-import { DataTableCellProps } from '@epam/uui-core';
+import { DataTableCellProps as UuiCoreDataTableCellProps, Overwrite } from '@epam/uui-core';
 import { DataTableCell as UuiDataTableCell } from '@epam/uui-components';
 import { DataRowAddons } from '../widgets';
-import { DataTableCellMods } from './types';
-import { TextPlaceholder, Text, TextProps } from '../typography';
+import type { DataTableCellMods } from './types';
+import { TextPlaceholder, Text } from '../typography';
 import { Tooltip } from '../overlays';
 import { settings } from '../../settings';
 
 import './variables.scss';
 import css from './DataTableCell.module.scss';
 
-export function DataTableCell<TItem, TId, TCellValue>(initialProps : DataTableCellProps<TItem, TId, TCellValue> & DataTableCellMods) {
+export interface DataTableCellModsOverride {}
```

Comment:
Need to think about this override interfaces, probably we should use here overrides from global DataTableMods interface to not force users to define overrides for such internal parts